### DISCUSSION TITLE
Docker container of yandex api

### DISCUSSION THREAD

**Discussion by RynorX0** [MarshalX/yandex-music-api#574] [Идеи]
Здравствуйте. Пишу на Java, судя по гуглу - лучше вашего варианта нет, однако, я хотел бы юзать ваш фреймворк, но сами понимаете, не соединить их.

Моя просьба:
- Налепить swagger для вашего проекта
- Собрать в docker container
- Запуллить
- Пошарить =)

**Comment by MarshalX** [comment]
Здравствуйте. Вы правы, лучше нет. А что по поводу вашей просьбы, то предлагаю сделать это вам самостоятельно. Ждём pull request

**Reply by RynorX0** [reply]
Я питон не знаю)
